### DISCUSSION TITLE
Asus G713-RS max fan RPM can be more?

### DISCUSSION THREAD

**Discussion by UgurInanc12** [seerge/g-helper#2183] [Ideas]
Hello everyone
First I want to thanks everyone who works on this project <3 

My question is my laptop's fan capable to turn at higher RPM (CPU fan is 6600rpm and GPU fan is 6700rpm). The way I found it was funny.
I tried fan test from MY ASUS program (in system diagnosis) and it start to turn my fans in much higher rpm's than regular armory crate or this G-HELPER

I wonder if somehow I can able to make my fans turn faster. It could be effect cooling system a lot.

in the picture I'm attached shows the higher RPM's for my fans because like I said before I start calibrating in G-HELPER while system diagnosis starts. Program reads higher RPM's and change the default ones. But it not working

![Screenshot 2024-03-08 213458](https://github.com/seerge/g-helper/assets/55364464/42658cc2-5fb9-43c0-a64c-efbc37bc8516)

Normally CPU fan should maxed at 5500rpm and GPU fan should maxed at 6000rpm

**Comment by seerge** [comment]
@UgurInanc12 it depends on how it's programmed in BIOS :) fan curve is being set in % under the hood. What 100% means - depends on specific model.

**Reply by UgurInanc12** [reply]
yes that's make sense. So your program or armory crate order to bios turn fans at %100 not the specific rpm.

But what I'm don't understand is how My Asus program can turn at much higher rpm. Is it ordering bios to turn fan at %120 or something like that maybe?

**Reply by seerge** [reply]
@UgurInanc12 MyASUS runs another command :) it's designed for testing fans (and checking if they work), not running them on a long term under fan curve. But it's just an ON / OFF command.
